requirements_bundles.txt

```diff
@@ -2,6 +2,7 @@
 # loading mechanism need on Python 2 and can be removed
 # when moved to Python 3.
 # It's automatically installed when running npm run bundle
+redash-stmo>=2019.9.0
```

Comment:
@emtwo I'm not sure why this is in this file, I probably missed this in a past rebase, but this shouldn't be in requirements_bundles.txt but in requirements.txt.